src/test/ui/privacy/associated-item-privacy-trait.stderr

```diff
@@ -31,7 +31,7 @@ LL |     priv_trait::mac!();
    |
    = note: this error originates in the macro `priv_trait::mac` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error: associated constant `<Pub as PrivTr>::CONST` is private
+error: associated constant `PrivTr::CONST` is private
```

Comment:
What caused this change? It feels like the previous output might be better (although if `CONST` is priv in the trait, it will in the impl too).

Reply:
That makes a lot of sense and it is great!